specs/DesignDiscussions/Inconsistencies.md

```diff
@@ -0,0 +1,169 @@
+* Supporting libraries and platform affordances available to us can ease or speed up implementation, but can also be the source of inconsistencies (e.g. differences in how a dev constructs .NET's `DateTime`, UWP's `DateTime`, C++'s `std::chrono::time_point`, and Javascript's `Date` -- instantiating each object with, for example, a string, can vary in permissivity)
+  * Indeed, we've had a few issues arise from differences between a hand-rolled solution and a pre-rolled solution (e.g. date/time handling (#1915) and support for markdown (#1984))
+
+### Rendering
+
+Rendering presents another avenue by which inconsistencies can sneak into Adaptive Cards. Some of these problems can be pretty tricky to solve.
+
+* Multiple implementations means that "gotcha" scenarios might not always be covered or may be covered inconsistently (e.g. what happens when you combine `maxLines`, markdown lists, and `wrap: true`?)
+* Platform-provided controls can vary widely in their behaviors and support. Even if divergent behavior is caught during implementation, it may be difficult or tricky to solve completely
+
+## Proposals
+
+* (P0) Change processes to prevent new inconsistencies from ever happening:
+  * Motivation: Inconsistencies are caught before they ship
+  * Tasks: 
+    * New bugs automatically filed with `inconsistency` tag, which is only removed once it's determined that the issue is *not* a consistency issue
+    * We can't ship if an `inconsistency`-tagged bug hasn't been resolved for this release
+    * Features that need implementation across all platforms are merged to a single branch and PR'd/tested for inconsistencies before being allowed to merge upstream
+    * Require that first accepted commits of a new feature include robust test cards that not only exercise the "golden path" for a feature, but every conceivable corner case relevant to 
+      * Perhaps a checklist that needs to be completed prior to merge?
+* (P1) Provide a new tool that allows easy testing of card payloads across all renderers:
+  * Motivation: Allows for the easy directed testing of feature consistency
+  * Tasks: 
+    * If shipped for public use, this could be a nice addition to the designer to allow card authors to preview their cards in a variety of renderers
+    * Could be used in tests for per-platform visual verification
+    * Per-platform accessibility trees could also be generated by this tool to allow for cross-plat verification
+* (P1) Make hard decisions around defaults, required properties, and parser/renderer behaviors: 
+  * Motivation: When these decisions are fully documented, it's easier for a line-level dev to choose correct behavior, meaning that it's more intuitive as to what the correct behavior should be.
+  * Tasks: 
+    * Document decisions publicly & in code as appropriate
```

Comment:
> Document decisions publicly & in code as appropriate [](start = 5, length = 53)

Maybe add a note here about updating the schema files in the repro along with the spec and/or code to ensure that decisions make it into the schema documentation for future reference.